source/nodejs/adaptivecards-designer/src/containers/outlook-container.ts

```diff
@@ -116,9 +116,9 @@ export class OutlookContainer extends HostContainer {
         }
     }
 
-    public anchorClicked(rootCard: Adaptive.AdaptiveCard, anchor: HTMLAnchorElement): boolean {
+    public anchorClicked(element: Adaptive.CardElement, anchor: HTMLAnchorElement): boolean {
         var regEx = /^action:([a-z0-9]+)$/ig;
-
+        var rootCard = element.getRootElement(); 
```

Comment:
We should check that this is an AdaptiveCard. getRootElement returns a CardElement.